src/components/dev-hub/cms-form.js

```diff
@@ -0,0 +1,90 @@
+import React, { useReducer } from 'react';
+import Checkbox from '@leafygreen-ui/checkbox';
+import Input from './input';
+import TextArea from './text-area';
+
+const getInitialFormState = () => ({});
+
+const FORM_ELEMENT_TYPES = {
+    CHECKBOXES: 'Checkboxes',
+    EMAILINPUT: 'EmailInput',
+    TEXTAREA: 'Textarea',
+};
+
+const handleCMSFormChange = (state, { field, value }) => ({
+    ...state,
+    [field]: value,
+});
+
+const mapTypeToFormElement = (type, labels, state, dispatch, name) => {
+    switch (type) {
+        case FORM_ELEMENT_TYPES.CHECKBOXES:
+            return labels.map(({ label }, index) => (
+                <Checkbox
+                    key={`${name}-${index}`}
+                    bold
+                    onChange={e =>
+                        dispatch({
```

Comment:
Looks like dispatch for checkbox always return 'on' even if checkbox unchecked. Try to check and after uncheck checkbox - e.target.value will be 'on' for two cases.

Reply:
Good point, I fixed this to use the `checked` field instead of `value` so this should work as expected.